assets/src/setup/components/stepper/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useInstanceId } from '@wordpress/compose';
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { Check } from '../svg/check';
+
+/**
+ * List bullet for a stepper item.
+ *
+ * @param {Object} props Component props.
+ * @param {number} props.activePageIndex The index of the currently visible screen.
+ * @param {number} props.index The index of the item being rendered.
+ */
+export function StepperBullet( { activePageIndex, index } ) {
+	const isCheckMark = useMemo( () => index === 0, [ index ] );
+	const isCurrent = useMemo( () => ( ! isCheckMark ) && activePageIndex === index, [ activePageIndex, index, isCheckMark ] );
```

Comment:
Those two calculations are so simple and fast that the overhead of using `useMemo()` will not be offset by the gain. The values should just be calculated here.
```suggestion
	const isCheckMark = index === 0;
	const isCurrent = ( ! isCheckMark ) && activePageIndex === index;
```

Reply:
Agreed, updating. These were more complicated at an early stage of development